mqtt/mqtt-bridge/config/default.json

```diff
@@ -1,4 +1,13 @@
 {
-    "subscriptions": [],
-    "forwards": []
+    "upstream":
+    {
+        "keep_alive": "1m",
+        "clean_session": true,
+        "subscriptions": [],
+        "forwards": []
+    },
+    "remotes": [],
+    "messages": {
+        "max_count": 100
```

Comment:
This is max messages in memory? As in read from disk at a time? Why is this needed as a setting? (I see it in the doc, just curious).

Reply:
iirc it is maximum inflight messages settings. it should not be that big number. Broker has 16 now

Reply:
from the spec this is number of messages per topic bridge stores in memory which I understand as how many are read from disk at a time, not inflight messages. It should be renamed to make it clear what it refers to.

Reply:
I removed max_count for now since it was not clear what was needed for and will add it as needed.